packages/runtime/container-runtime/src/summaryFormat.ts

```diff
@@ -149,3 +152,24 @@ export function wrapSummaryInChannelsTree(summarizeResult: ISummaryTreeWithStats
     };
     summarizeResult.stats.treeNodeCount++;
 }
+
+export async function getFluidDataStoreAttributesFromStorageOrSnapshot(
+    storage: IDocumentStorageService,
+    snapshot: ISnapshotTree,
+): Promise<ReadFluidDataStoreAttributes> {
```

Comment:
I find name of this type weird. The "Read" part. I'd drop it. I think it would be consistent with what it is:

export type FluidDataStoreAttributes =
    | IFluidDataStoreAttributes0
    | IFluidDataStoreAttributes1
    | IFluidDataStoreAttributes2;

Reply:
I think there is "WriteFluidDataStoreAttributes" too. So that is why we have "ReadFluidDataStoreAttributes" imo.